opendcs-rest-api/src/test/resources/tomcat/conf/tomcat-server.xml

```diff
@@ -0,0 +1,60 @@
+<!--
+  ~  Copyright 2024 OpenDCS Consortium and its Contributors
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License")
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+<!--
+-->
+<Server port="-1" shutdown="SHUTDOWN">
+    <Listener className="org.apache.catalina.startup.VersionLoggerListener"/>
+    <!--
+    APR library loader. Documentation at /docs/apr.html
+    -->
+    <Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on"/>
+    <!--
+     Prevent memory leaks due to use of particular java/javax APIs
+    -->
+    <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener"/>
+    <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener"/>
+    <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener"/>
+    <GlobalNamingResources> </GlobalNamingResources>
+    <!--
+     A "Service" is a collection of one or more "Connectors" that share
+           a single "Container" Note:  A "Service" is not itself a "Container",
+           so you may not define subcomponents such as "Valves" at this level.
+           Documentation at /docs/config/service.html
+
+    -->
+    <Service name="Catalina">
+        <Engine name="Catalina" defaultHost="localhost">
+            <Host name="localhost" appBase="webapps" unpackWARs="true" autoDeploy="true">
+                <!--
+                SingleSignOn valve, share authentication between web applications
+                        Documentation at: /docs/config/valve.html
+                -->
+                <!--<Valve className="fixtures.TestAuthValve"/>-->
+                <Valve className="org.apache.catalina.authenticator.SingleSignOn"/>
+                <Valve className="org.apache.catalina.valves.ErrorReportValve"
+                    showReport="false" 
```

Comment:
These can, and perhaps should, be true for test runs.

Reply:
updated